packages/vite-plugin-cloudflare/playground/dev-vars/package.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "@playground/dev-vars",
+	"private": true,
+	"type": "module",
+	"scripts": {
+		"build": "vite build --app",
+		"check:types": "tsc --build",
+		"dev": "vite dev",
+		"prepreview": "node ./copy-dev-vars-to-dist.mjs",
```

Comment:
Is this still needed?

Reply:
damn, no sorry this was for some temporary hacking, thanks a lot for noticing it 🙏